current/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Current Litter | Doodle Dandy</title>
+    <link rel="stylesheet" href="../assets/vendor/simplecss/simple.min.css" />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/fontawesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/brands.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/solid.min.css"
+    />
+    <link rel="stylesheet" href="../assets/styles/main.css" />
+    <link
+      rel="shortcut icon"
+      href="../assets/images/favicon.ico"
+      type="image/x-icon"
+    />
+  </head>
+
+  <body>
+    <header>
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
+        <a href="/famstories">Reviews</a>
+        <a href="/contact">Contact us</a>
+      </nav>
+    </header>
+    <main>
+      <h1>Current Litter | Doodle Dandy</h1>
```

Comment:
Don't use the `|` character like this in the actual page content.
```suggestion
      <h1>Current Litter</h1>
```

Reply:
FIXED

Reply:
sorry cap lock was on I did not notice